staging/dex-k8s-authenticator/html-templates/kubectl-plugin-instructions.html

```diff
@@ -0,0 +1,112 @@
+<h1>Build kubeconfig with kubectl</h1>
+{{ if .caPem }}
+    <h2>Copy Cluster CA Certificate</h2>
+    <p>This is the CA certificate needed to validate connections to your cluster. If your cluster is using a well known
+        certificate authority, such as Let's Encrypt, or you have added your CA certificate to your system certificate pool;
+        you may skip this section.</p>
+    <div class="command">
+
+        <button class="btn" style="float:right" data-clipboard-snippet="">
+            <img class="clippy" width="13" src="{{ .webPrefixPath }}static/clippy.svg" alt=""/>
+        </button>
+        <pre><code>mkdir -p ${HOME}/.kube/certs/{{ .clusterName }}/ &amp;&amp; cat &lt;&lt; EOF &gt; ${HOME}/.kube/certs/{{ .clusterName }}/k8s-ca.crt
+{{ .caPem }}
+EOF</code></pre>
+    </div>
+{{ end }}
+
+<h2>Download and Install the Konvoy credentials plugin</h2>
```

Comment:
Title Case or not title case.

```suggestion
<h2>Download And Install The Konvoy Credentials Plugin</h2>
```